ssg-launch-group.c

```diff
@@ -199,18 +200,25 @@ int main(int argc, char *argv[])
     if (opts.gid_file)
         ssg_group_id_store(opts.gid_file, g_id, SSG_ALL_MEMBERS);
 
-    /* sleep for given duration to allow group time to run */
-    if (opts.shutdown_time > 0)
-        margo_thread_sleep(mid, opts.shutdown_time * 1000.0);
+    /* sleep a bit for group to stabilize */
+    margo_thread_sleep(mid, 5 * 1000.0);
```

Comment:
not a blocker for this code, but reminds me that we don't have any better way to detect "no problems, but things are still negotiating with each other" -- so many job scripts with arbitrary "sleep 30" in them.

Reply:
For cases like this it might be helpful to just have a little subroutine that can wait until all members are in the up state or something?  But yeah, these arbitrary time things are a little sketchy.